src/games/strategy/triplea/ai/proAI/data/ProTerritory.java

```diff
@@ -432,4 +435,12 @@ public void setMaxScrambleUnits(final List<Unit> maxScrambleUnits) {
   public List<Unit> getMaxScrambleUnits() {
     return maxScrambleUnits;
   }
+
+  public List<Unit> getBombers() {
+    return bombers;
+  }
+
+  public void setBombers(final List<Unit> bombers) {
```

Comment:
AI adds bombers one by one to the list. This tracks which bombers are going to bomb which territories.